packages/odyssey-react-mui/src/@types/react-augment.d.ts

```diff
@@ -10,14 +10,18 @@
  * See the License for the specific language governing permissions and limitations under the License.
  */
 
-import { FC } from "react";
-
+import "react";
```

Comment:
Any reason why you removed `FC` as an imported object? I believe this affects the interface definition below where it extends `FC`

Reply:
I believe this should be something like `import React, { FC } from "react";`

Reply:
@chrispulsinelli-okta So this change to `react-augment.d.ts` came from a discussion with Kevin. I saw that you were the original author of this file but Kevin was saying he thought the `ForwardRefWithType` definition would actually fit better in `inputUtils.ts` (just like what I did with `FocusHandle`) since [`d.ts` files shouldn't be used for holding custom types](https://github.com/microsoft/TypeScript/issues/52593#issuecomment-1419505081). I'm not a TS expert so I'm most relaying what Kevin was saying, but he originally wanted me to move your `ForwardRefWithType` interface into `inputUtils.ts` in this ticket as well but it led to a bunch of TS compiler errors when I tried to do so since Odyssey has `"skipLibCheck": true` in `tsconfig` which prevents type checking in `d.ts` files. 

I don't have any context behind `ForwardRefWithType` and didn't want to break your code so Kevin asked me to create a ticket and assign it to you for this sprint: https://oktainc.atlassian.net/browse/OKTA-688308

Regardless, I can still add back the `FC` for the scope of this PR